internal/log/log.go

```diff
@@ -12,8 +12,6 @@ var (
 	Info    *log.Logger
 	Warning *log.Logger
 	Error   *log.Logger
-	Fatal   *log.Logger
-	Stdout  *log.Logger
```

Comment:
I decided to remove some extra loggers as part of this PR, so there will be some seemingly random log lines update in the diff